tests/Fixtures/TestBundle/Entity/Foo.php

```diff
@@ -24,7 +23,7 @@
  * @author Vincent Chalamon <vincentchalamon@gmail.com>
  *
  * @ApiResource(attributes={
- *     "order"={"name", "DESC"}
+ *     "order"={"bar": "ASC", "name": "DESC"}
```

Comment:
@dunglas Could it also be interesting to order by association property, like `dummy.name`?